src/RoomsSDKAdapter.js

```diff
@@ -134,4 +144,119 @@ export default class RoomsSDKAdapter extends RoomsAdapter {
 
     return this.getRoomObservables[ID];
   }
+
+  /**
+   * Invoke request to listActivities on adapter
+   *
+   * @param {object} Params to send to conversation api method
+   * @returns
+   * @private
+   */
+  fetchActivities({roomId, earliestActivityDate}) {
```

Comment:
Let's have this explicit arguments instead of an object
```suggestion
  fetchActivities(roomID, earliestActivityDate) {
```

Reply:
Deconstructed object is pretty explicit, also makes the API extendable if we wanted to add more options to the underlying request, using object parameters is a best practice as we do not enforce developers to remember the order of parameters or code changes when we add additional required/optional parameters.